### ISSUE TITLE
Set up OCW QA instances in AWS

### ISSUE THREAD

**Issue by shaidar** [mitodl/salt-ops#671]
Set up and configure OCW QA instances in AWS.

**Comment by shaidar** [comment]
After looking at the current QA setup and discussing with Tobias, decided to clone `ocw-production-cms-1` and `ocw-production-cms-db1` and make the needed changes to get QA instances in AWS up and running. Am currently in the process of configuring the instances.

**Comment by shaidar** [comment]
Scheduled to stop services on `ocwqa` and `ocwqadb` tonight and start syncing files to the AWS instances.

**Comment by shaidar** [comment]
Files from qa have been synced to AWS QA instances. Still need to configure a few settings and compile python with openssl prior to considering switching off to using that AWS setup and shutting down existing QA.

**Comment by pdpinch** [comment]
Is this still part of the plan for migrating OCW to AWS?

**Comment by blarghmatey** [comment]
It is part of the eventual plan, but not a hard requirement for cutting over.

**Comment by shaidar** [comment]
QA instances up and running in AWS. Waiting on Joe to confirm functionality.

**Comment by blarghmatey** [comment]
@shaidar what's the current status of this? I believe that Joe has done at least some work on the QA instances.

**Comment by shaidar** [comment]
@blarghmatey I think he looked around a bit, but not sure to what extent he tested things out. Will follow up with him to get things moving a bit on this front.

**Comment by pdpinch** [comment]
@markbreedlove what remains to be done here? Maybe a full-site republish??

**Comment by markbreedlove** [comment]
I'd thought so, but now I think it's best to make smaller, more detailed tickets. I think you were saying something to that effect with another issue that was closed last week.

**Comment by markbreedlove** [comment]
I'll close this now so that we can make those smaller issues.
